File: Scripts/single_notification.php
<?php

function sendFCM($id) {
$url = 'https://fcm.googleapis.com/fcm/send';
$fields = array (
        'to' => $id,
        'data' => array (
                "notif1" => "test",
                "notif2" => "Title"
        )
);

$fields = json_encode ( $fields );
$headers = array (
        'Authorization: key=' . "_____",
        'Content-Type: application/json'
);

$ch = curl_init ();
curl_setopt ( $ch, CURLOPT_URL, $url );
curl_setopt ( $ch, CURLOPT_POST, true );
curl_setopt ( $ch, CURLOPT_HTTPHEADER, $headers );
curl_setopt ( $ch, CURLOPT_RETURNTRANSFER, true );
curl_setopt ( $ch, CURLOPT_POSTFIELDS, $fields );

$result = curl_exec ( $ch );
echo($result);
curl_close ( $ch );
}

sendFCM("____device ID___");

?>

File: Scripts/single_notification.py
import requests
import json
#http://docs.python-requests.org/en/master/user/quickstart/
MY_API_KEY="_____________"

datas={
    "to" : "______device id____",
    "data" : {
        "notif1" : "test",
        "notif2" : "test2"
    }
}
dataAsJSON = json.dumps(datas)



headers = { "Authorization" : "key="+MY_API_KEY,
            "Content-type" : "application/json"
            }

r = requests.post('https://fcm.googleapis.com/fcm/send', data = dataAsJSON, headers=headers )

print(r.text)
